website/content/v1.8/advanced/install-kubevirt.md

```diff
@@ -0,0 +1,401 @@
+
+When you want to leverage [Multus](../kubernetes-guides/network/multus) to give your virtualm achines direct access to your nodes network, your bridge need to configured properly. This can be done by setting your network interface in bridge mode. You can look up the network interface name by using the following command:
+
+```bash
+$ talosctl get links -n 10.99.101.9
+NODE          NAMESPACE   TYPE         ID             VERSION   TYPE       KIND     HW ADDR                                           OPER STATE   LINK STATE
+10.99.101.9   network     LinkStatus   bond0          1         ether      bond     52:62:01:53:5b:a7                                 down         false
+10.99.101.9   network     LinkStatus   br0            3         ether      bridge   bc:24:11:a1:98:fc                                 up           true
+10.99.101.9   network     LinkStatus   cni0           9         ether      bridge   1e:5e:99:8f:1e:19                                 up           true
+10.99.101.9   network     LinkStatus   dummy0         1         ether      dummy    62:1c:3e:d5:72:11                                 down         false
+10.99.101.9   network     LinkStatus   eth0           5         ether               bc:24:11:a1:98:fc                         
+```
+
+In this case, this network interface is called `eth0`. Now you can configure your bridge properly. This can be done in the machine config of your node:
+
+```yaml
+machine:
+      interfaces:
+      - interface: br0
+        addresses:
+          - 10.99.101.9/24
+        bridge:
+          stp:
+            enabled: true
+          interfaces:
+              - eth0 # This must be changed to your matching interface name
+        routes:
+            - network: 0.0.0.0/0 # The route's network (destination).
+              gateway: 10.99.101.254 # The route's gateway (if empty, creates link scope route).
+              metric: 1024 # The optional metric for the route.
+``` 
+
+### Install the `local-path-provisioner`
+
+When we are using KubeVirt, we are also installing the CDI (containerized data importer) operator. For this to work properly, we have to install the `local-path-provisioner`. This CNI kan be used to write scratch space when importing images with the CDI. 
+
+You can simply install the `local-path-provisioner` by following [this](../kubernetes-guides/configuration/local-storage) guide
+
+### Configure storage
+
+If you would like to use features such as `LiveMigration` shared storage is neccesary. You can either choose to install a CSI that connects to NFS or you can install Longhorn, for example. For more information on how to install Longhorn on Talos you can follow [this](https://longhorn.io/docs/1.7.2/advanced-resources/os-distro-specific/talos-linux-support/) link.
+
+To install the NFS-CSI driver, you can follow [This](https://github.com/kubernetes-csi/csi-driver-nfs/blob/master/docs/install-csi-driver-v4.9.0.md) guide. 
```

Comment:
What's the difference between this link and the one above?

Reply:
The first one is the installation of Longhorn on Talos. The other one is the installation of the NFS-CSI driver. Which is neccesary if you want to connect to a NFS share.